docs/kits/Business Partner Kit/Documentation BPDM/page_troubleshooting.md

```diff
@@ -0,0 +1,21 @@
+---
+id: Troubleshooting
+title: Troubleshooting
+description: ''
+sidebar_position: 5
+---
+1. Invalid input data: This error occurs when the data submitted to the server is not in the correct format. To resolve this issue, validate the data and ensure that it conforms to the expected format.
+
+    Example - When an error occurs, provided a clear and informative error message to the user. The error message is easy to understand and provide enough information for the user to take corrective action. For example, if the user enters an invalid email address, the error message is something like "Invalid email address. Please enter a valid email address".
+
+2. Authentication errors: These errors occur when the user is not authorized to access the service. To resolve this issue, ensure that the user has the appropriate permissions and that the authentication credentials are correct.
+
+    Example - If a user tries to use api services provided without authentication in, the server return an error message that states "401 Unauthorized."
+
+3. Server errors: These errors occur when there is a problem with the server. To resolve this issue, check the server logs and contact the server administrator.
```

Comment:
Also here sections should be divided into whether you are an admin or just use an API. In one case you can check the server logs and in the other case you need to contact an admin. Separate those use cases and provide troubleshooting advice for each separately